packages/ie11-custom-properties/ie11CustomProperties.js

```diff
@@ -0,0 +1,675 @@
+/*! ie11CustomProperties.js v4.1.0 | MIT License | https://git.io/fjXMN */
```

Comment:
Should probably remove this line, or replace it with something indicating the original source.

Reply:
I have removed this line